### ISSUE TITLE
Alpha 2 doesnt start.

### ISSUE THREAD

**Issue by holycow131415** [burst-apps-team/burstcoin#43]
Alpha 1 starts just fine with no configuration changes. Unpacking the zip for Alpha2, and launching burst.cmd with no config changes yield this.

![image](https://user-images.githubusercontent.com/25784532/52249339-9202ce00-28c0-11e9-91b5-57d19001af08.png)

**Comment by harry1453** [comment]
This is an intentional check. Alpha2 is not a tested stable build, so can only be used on testnet. If you configure the node to run on the testnet, it will work.

**Comment by holycow131415** [comment]
Thx I'll try later, The release notes of Alpha2 say "The attached ZIP contains a default configuration for running on the TestNet." I also tried finding instructions in the Readme to set up being on testnet but its not there. Maybe it meant the source code zip and not the alpha 2 zip? 

After typing the above, I had to really search hard for https://github.com/burst-apps-team/BurstcoinTestNetHow-To so can I suggest to delete this and put it in the wiki here or the readme in this project?
